### ISSUE TITLE
Invalid News Article (Maybe Security Feature)

### ISSUE THREAD

**Issue by CodeLaws** [FusionGen/FusionGEN#231]
Here you can see how my article looks in the text editor.

![925922](https://i.imgur.com/XlRZEbh.png)

But in Browser it looks like this

![https://i.imgur.com/594jMYh.png](https://i.imgur.com/594jMYh.png)

Seems to me a security feature inside the project is blocking from showing it correctly?

**Comment by CraftedRO** [comment]
what is invalid here, I only see the yellow text not being yellow

**Comment by CodeLaws** [comment]
yep, this is invalid.

**Comment by Ferreira9006** [comment]
Delete all the cache (FusionGEN / Browser cache) as that could be causing the issue.

If that doesn't fix, please verify on the database that the editor is actually applying the color on the text.

**Comment by CraftedRO** [comment]
yeah the issue is still valid: create news article>select a word and then use Bold and then yellow text color, notice on news the color was not applied.

**Comment by CraftedRO** [comment]
after some research it seems that this theme css style does overwrite some color formatting sometimes:

> \application\themes\default\assets\css\parts\page.css L109

> .pagebody *    {color: #5f758a;}

I'm not sure how we'll fix this since if we change this, text color in other pages will look bad because of bootstrap default.
